cpp/util/etcd.cc

```diff
@@ -668,10 +680,36 @@ bool EtcdClient::Node::HasExpiry() const {
 
 
 EtcdClient::EtcdClient(UrlFetcher* fetcher, const string& host, uint16_t port)
-    : fetcher_(CHECK_NOTNULL(fetcher)), endpoint_(host, port) {
-  CHECK(!endpoint_.first.empty());
-  CHECK_GT(endpoint_.second, 0);
+    : EtcdClient(fetcher, list<HostPortPair>{HostPortPair(host, port)}) {
+}
+
+
+EtcdClient::EtcdClient(UrlFetcher* fetcher, const list<HostPortPair>& etcds)
+    : fetcher_(CHECK_NOTNULL(fetcher)), etcds_(etcds) {
+  CHECK(!etcds_.empty());
```

Comment:
I know I didn't have them originally, but some explanation strings on those startup-time `CHECK`s would be nice.